setup.py

```diff
@@ -28,8 +28,6 @@
         'flake8!=2.5.3',
         'autopep8>=1.3',
         'pyyaml',
-        'simplejson',
-        'six',
```

Comment:
six is used elsewhere so it can't be removed

Reply:
I missed that other usage, that can be easily resolved too. :)

Reply:
tbh, until we're comfortable dropping python 2 entirely, keeping six around seems ok to me -- I'd certainly rather see `from six.moves import ...` than `try: except:` for imports

Reply:
For a single import to require six, I'd disagree as it creates an extra level of abstraction to a 3rd part module and a docs lookup for usage. With a try..except you know exactly what is going on. Both will have to be modified in future regardless though I suppose it'd then be easier to find the six module imports...